src/App.tsx

```diff
@@ -1,56 +1,87 @@
-import React from 'react';
+import React, { useState, useMemo } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 import { peopleFromServer } from './data/people';
+import { PeopleList } from './components/PeopleList';
+import { Person } from './types/Person';
+import { SearchField } from './components/SearchField';
+
+const findPersonBySlug = (personSlug: string): Person | null => {
+  return peopleFromServer.find(person => person.slug === personSlug) || null;
+};
+
+const getPeopleByQuery = (people: Person[], query: string): Person[] => people
+  .filter(person => {
+    const personName = person.name.toLowerCase();
+
+    return personName.includes(query.toLowerCase());
+  });
```

Comment:
I would say this way is clear enough
```suggestion
const getPeopleByQuery = (
  people: Person[], 
  query: string
): Person[] => (
  people.filter((person) => (
    person.name.toLowerCase()
      .includes(query.toLowerCase())
  ));
)
```